misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import pytz
+
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils.six import StringIO
+from django.utils import timezone
+
+from misago.conf import settings
+from misago.users.management.commands import anonymizeoldips
+from misago.core.utils import ANONYMOUS_IP
+
+
+# - command is not anonymizing user's IP if anonymization is disabled
+# - command is anonymizing user's IP if its old
+
+UserModel = get_user_model()
+
+
+class AnonymizeOldIpsTests(TestCase):
+    def setUp(self):
+        self.user = UserModel.objects.create_user('Bob', 'bob@bob.com', 'pass123')
```

Comment:
Please don't create those accounts in `setUp`. Each test case should have its own `user = UserModel.objects.create(...`.

Since Misago 0.18 you also don't need to specify password for user created with `create_user`. `create_user('Bob', 'bob@bob.com')` is fine too.